wiktextract/clean.py

```diff
@@ -1381,8 +1381,8 @@ def repl_1_syntaxhighlight(m):
     # Change <div> and </div> to newlines.  Ditto for tr, li, table, dl, ul, ol
     title = re.sub(r"(?si)</?(div|tr|li|table|dl|ul|ol)\b[^>]*>",
                    "\n", title)
-    # Change </dt> and </dd> into newlines; these generate new rows/lines.
-    title = re.sub(r"(?si)</(dt|dd)\s*>", "\n", title)
+    # Change <dt> and <dd> into newlines; these generate new rows/lines.
+    title = re.sub(r"<d[dt]>", "\n", title)
```

Comment:
The whitespace at the end of a HTML tag is valid HTML (not at the start), so we have to keep the `\s*` in case someone does type `<dt >`.

Reply:
I think both the start and end tags could be turned into newlines, just in case. `r"</?d[dt]\s*>`

Reply:
I'll just do it quick-like.

Reply:
Thanks!